content/docs/faq-state.md

```diff
@@ -0,0 +1,59 @@
+---
+id: faq-state
+title: Component State
+permalink: docs/faq-state.html
+layout: docs
+category: FAQ
+---
+
+### What does setState do?
+
+`setState()` schedules an update to a component's `state` object. When state changes, the component responds by re-rendering.
+
+### Why is `setState` is giving me the wrong value?
+
+Calls to `setState` are batched, so it is possible to "lose" an update if you call it with the partial object syntax:
```

Comment:
I'm not sure the wording "_partial object syntax_" is particularly user-friendly. I'm not sure this is much better, but what do you think about this?

> Calls to `setState` may be asynchronous! If you need to reference a value in state when updating it, use an [updater function](/docs/react-component.html#setstate) instead. ([See below](#what-is-the-difference-between-passing-an-object-or-a-function-in-setstate) for an example.)

Reply:
Good point about that phrasing; I'd like to come up with something in between that explains a bit more about why async patches can drop updates. A link to a Codepen might be sufficient.